recipes/aicspylibczi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aicspylibczi" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+    sha256: d903caaac5576922f8dd4b1170fae3421fb9d203128b4d8ecb11681173e7046c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    # - libpng
+    - python
+    - pip
+    - pytest-runner
+    # - zlib
```

Comment:
@toloudis is this still needed?

Reply:
> @toloudis is this still needed?

Honestly not sure. If I had to guess I would say no, not needed.

Reply:
I ask because of what the readme says

https://github.com/AllenCellModeling/aicspylibczi#build


>Build
>Use these steps to build and install aicspylibczi locally:

>Clone the repository including submodules (--recurse-submodules).
>Requirements:
>libCZI requires a c++11 compatible compiler. Built & Tested with clang.
>Development requirements are those required for libCZI: libpng, zlib

Reply:
I haven't confirmed it but the only place our repo explicitly refers to those libs is here: 
https://github.com/AllenCellModeling/aicspylibczi/blob/2e2faeefa1e3fea3b0da979db07f85c6be746904/.github/workflows/py-build-main.yml#L106

I believe that might be outdated due to this https://github.com/ZEISS/libczi/blob/134582181b5e121c4a1b0236ae707e2db36f66ff/Src/libCZI/Doc/building_libCZI.markdown?plain=1#L58 
(`libpng` and `zlib` only needed for `czicmd` build target which is not required for `aicspylibczi` ).

However, it doesn't hurt to keep them in?

Reply:
👍🏼